### DISCUSSION TITLE
is it possible to inject more CLLocation parameters.

### DISCUSSION THREAD

**Discussion by rorlig** [doronz88/pymobiledevice3#788] [Q&A] [answered]
is it possible to inject more CLLocation parameters. 

the current api only allows for lat , lng. Can I send mock altitude, accuracy etc?

**Comment by doronz88** [comment]
that's all available via protocol

**Reply by rorlig** [reply]
can you elaborate - "that's all available via protocol" ? Does this mean it is not possible to do this or it is ? Your response was a bit unclear.

**Reply by rorlig** [reply]
also sorry for noob comment, what protocol ?

**Reply by doronz88** [reply]
the protocol messages the devices exposes and pymobile communicates with

**Reply by rorlig** [reply]
i see, 

so to clarify 

You are saying only lat/lng can be send ? 


Do you have link the protocol definition ?

**Reply by doronz88** [reply, marked as answer]
yes only them - and no, this is done by reversing the service

**Reply by rorlig** [reply]
any document on how to reverse a service ?
